epoxy-composesample/src/main/java/com/airbnb/epoxy/compose/sample/ComposableInteropActivity.kt

```diff
@@ -0,0 +1,86 @@
+package com.airbnb.epoxy.compose.sample
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Divider
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.airbnb.epoxy.EpoxyRecyclerView
+import com.airbnb.epoxy.composableInterop
+import com.airbnb.epoxy.compose.sample.epoxyviews.headerView
+
+class ComposableInteropActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_composable_interop)
+
+        val recyclerView: EpoxyRecyclerView = findViewById(R.id.epoxy_recycler_view)
+
+        recyclerView.withModels {
+            headerView {
+                id("header")
+                title("Testing Composable in Epoxy")
+            }
+
+            for (i in 0..100) {
+                composableInterop(
+                    id = "compose_text_$i",
+                    composeFunction = {
+                        ShowCaption("Caption coming from composabl")
+                    }
+                )
+
+                composableInterop(
+                    id = "news_$i",
+                    composeFunction = { NewsStory() }
+                )
```

Comment:
I think a more canonical usage of this would use trailing lambdas
```suggestion
                composableInterop(id = "news_$i"){
                    NewsStory()
                }
```

Reply:
yea my bad, wanted to change this.